### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by obwansan**
Built a portfolio website including a home page, profile page and portfolio page.

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.ie {
+    /* uiGradient - Telegram */
+    background: #1c92d2;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #f2fcfe, #1c92d2) fixed;  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #f2fcfe, #1c92d2) fixed; /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+.name {
+    margin-top: 300px;
+    text-align: center;
```
Comment (charliecog) [inline home.css]: put the text align on the parent div

File: home.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    font-size: 3rem;
+}
+
+.title {
+    margin-top: 5px;
+    text-align: center;
```
Comment (charliecog) [inline home.css]: same as .name class - put text align on parent so only have to declare it once

File: index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+</head>
+<!--[if IE]><body class="ie"><![endif]-->
+<body>
+    <div class="container">
+        <div class="text">
+            <div class="name">Kevin O'Brien</div>
+            <div class="title">Web Developer</div>
+        </div>
+        <div class="icons">
+            <a href="profile.html"><img class="page-link" src="images/profile-icon-1.png"</a>
```
Comment (charliecog) [inline index.html]: missing closing > from img tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            <a href="portfolio.html"><img class="page-link" src="images/portfolio-icon-1.png"</a>
```
Comment (charliecog) [inline index.html]: missing closing > from img tag

File: portfolio.css [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    left: 0;
+}
+
+/****************************
+Media Queries
+/****************************/
+
+@media screen and (max-width: 1340px) {
+
+    .col-4 {
```
Comment (charliecog) [inline portfolio.css]: col-4 with a width of 33.3% is misleading
create new md and sm class to handle media queries
eg: col-md-3

File: portfolio.css [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+        width: 33.3%;
+        margin-top: 10px;
+        margin: 0 auto;
```
Comment (charliecog) [inline portfolio.css]: this is overriding the previous margin top declaration

File: portfolio.html [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+            </div>
+            <div class="col-4">
+                <a href="#"><img class="portfolio-image" src="images/project-placeholder.png"></a>
+            </div>
+            <div class="col-4">
+                <a href="#"><img class="portfolio-image" src="images/project-placeholder.png"></a>
+            </div>
+        </div>
+        <div class="icons">
+            <a href="index.html"><img class="page-link" src="images/home-icon-1.png"</a>
```
Comment (charliecog) [inline portfolio.html]: img closing tag >

File: portfolio.html [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+            <a href="profile.html"><img class="page-link" src="images/profile-icon-1.png"</a>
```
Comment (charliecog) [inline portfolio.html]: img closing tag >

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,33 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: 'Josefin Sans', sans-serif;
+    color: #FFFFFF;
```
Comment (charliecog) [inline styles.css]: lowercase

**Review by charliecog — CHANGES_REQUESTED** [review]
nicely done, few changes but looks ok

File: home.css (hunk 1 above)
Comment (obwansan) [reply home.css]: done

File: home.css (hunk 2 above)
Comment (obwansan) [reply home.css]: done

File: index.html (hunk 3 above)
Comment (obwansan) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (obwansan) [reply index.html]: Done

File: portfolio.css (hunk 6 above)
Comment (obwansan) [reply portfolio.css]: fixed

File: portfolio.html (hunk 7 above)
Comment (obwansan) [reply portfolio.html]: fixed

File: portfolio.html (hunk 8 above)
Comment (obwansan) [reply portfolio.html]: fixed

File: styles.css (hunk 9 above)
Comment (obwansan) [reply styles.css]: done

File: portfolio.css (hunk 5 above)
Comment (obwansan) [reply portfolio.css]: I've googled a lot about how to do this but can't figure it out, so as a workaround I renamed "col-4" to "grid-column" which is less misleading. Can you let me know how to replace one class with another in a media query? (e.g. replace col-lg-4 with col-md-3). I couldn't work out how to stop col-lg-4 applying and make col-md-3 apply with a media query. Thanks

**Review by charliecog — APPROVED** [review]
looks great, all approved! feel free to complete the pull request! 
remember to git pull origin master from your local master branch to see the changes

**Pull request merged**